src/ParseUser.js

```diff
@@ -761,6 +783,69 @@ class ParseUser extends ParseObject {
     );
   }
 
+  /**
+   * Request an email verification.
+   *
+   * <p>Calls options.success or options.error on completion.</p>
+   *
+   * @param {String} email The email address associated with the user that
+   *     forgot their password.
+   * @param {Object} options
+   * @static
+   * @returns {Promise}
+   */
+  static requestEmailVerification(email: string, options?: RequestOptions) {
+    options = options || {};
+
+    const requestOptions = {};
+    if (options.hasOwnProperty('useMasterKey')) {
+      requestOptions.useMasterKey = options.useMasterKey;
+    }
+
+    const controller = CoreManager.getUserController();
+    return controller.requestEmailVerification(email, requestOptions);
+  }
+
+  /**
+   * Verify whether a given password is the password of the current user.
+   *
+   * @param {String} username  A username to be used for identificaiton
+   * @param {String} password A password to be verified
+   * @param {Object} options
+   * @static
+   * @returns {Promise} A promise that is fulfilled with a user
+   *  when the password is correct.
+   */
+  static verifyPassword(username: string, password: string, options?: RequestOptions) {
```

Comment:
Is using just `username` to verify password good enough? When I took a look on PS, I saw that it had supported both `email` and `username`.

Reply:
Can you write integration tests? I see you already have the unit tests.